packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -77,6 +79,7 @@ function DocSidebarItemCategory({
   // If we navigate to a category, it should automatically expand itself
   useEffect(() => {
     const justBecameActive = isActive && !wasActive;
+    setEnableCSSStyles(false);
```

Comment:
this means the server rendered page will have the cssonly mode. It would lead to a jank on first page load.